tests/ft_locales/ft_locales.cpp

```diff
@@ -4946,6 +4947,8 @@ void Ft_Locales::checkAvailableLocales()
     QCOMPARE(uint(bytesWritten), qstrlen(ft_localesTestOutput.toUtf8().constData()));
     ft_localesTestOutputFile.close();
 
+// Current version of libmlocales does not include input-icu-52.txt tests, so disabling this for now
```

Comment:
52 would be hard coding the expected version.

 How about just turning QFAIL on opening failure to QSKIP?